docs/_docs/windows.md

```diff
@@ -43,65 +45,67 @@ Now all that is left to do is install Jekyll.
 sudo gem install jekyll bundler
 ```
 
-Check to see if your installation worked.
+Check if Jekyll installed properly by running:
 
 ```
 jekyll -v
 ```
 
 **And that's it!**
 
-To start a new project, just run:
+To start a new project named `my_blog`, just run:
+
 ```
-jekyll new my_project
+jekyll new my_blog
 ```
 
-replacing `my_project` with the name of your website.
+You can make sure time management is working properly by inspecting your `_posts` folder. You should see a markdown file with the current date in the filename.
+
+**Note:** Bash on Ubuntu on Windows is still under development, so you may run into issues.
+
+
+[WSL-Guide]: https://msdn.microsoft.com/en-us/commandline/wsl/install_guide
+[BASH-WSL]: https://msdn.microsoft.com/en-us/commandline/wsl/about
+
+
+### Installation via RubyInstaller
 
-If you `cd` into the folder, you can make sure time management is working by opening your `_posts` folder. You should see a markdown file with the current date listed.
+[RubyInstaller][] is a self-contained Windows-based installer that includes the Ruby language, an execution environment, important documentation, and more.
 
-*Please note* Bash on Ubuntu on Windows is still under development, so you may run into issues. If you see an Auto-Regeneration error warning in your Bash instance, you can ignore it. 
+1. Download and Install a package manager version from [RubyInstaller Downloads][RubyInstaller-downloads].
+2. Install Jekyll and Bundler via a command prompt window: `gem install jekyll bundler`
+3. Check if Jekyll installed properly: `jekyll -v`
+
+[RubyInstaller]: https://rubyinstaller.org/
+[RubyInstaller-downloads]: https://rubyinstaller.org/downloads/
 
-## Installation via Chocolatey
 
+### Installation via Chocolatey
 
 A quick way to install Jekyll using Chocolatey is to follow the [installation instructions by David Burela](https://davidburela.wordpress.com/2015/11/28/easily-install-jekyll-on-windows-with-3-command-prompt-entries-and-chocolatey/):
 
- 1. Install a package manager for Windows called [Chocolatey](https://chocolatey.org/install)
+ 1. Install a package manager for Windows called [Chocolatey][]
```

Comment:
True. I kept it so that my text-editor applies the proper highlighting under `Markdown` mode. :wink: